providers/github.go

```diff
@@ -198,6 +206,43 @@ func (p *GitHubProvider) hasOrgAndTeam(accessToken string) (bool, error) {
 	return false, nil
 }
 
+// ValidateUser validates that the user exists as a collaborator on the specified repo
+func (p *GitHubProvider) ValidateUser(user string) bool {
+	//https://developer.github.com/v3/repos/collaborators/#check-if-a-user-is-a-collaborator
+
+	if p.Repo == "" {
+		return true
+	}
+
+	if p.OwnerToken == "" {
```

Comment:
Can we check this when we validate the flags rather than during runtime? This way we can return a reasonable error at startup rather than panicking at a later point

Reply:
fixed